Acl example

saflamini opened the pull request:
I want to add this simple acl example to this repo.

Smart contract automations are an important primitive, and we should have this here for the sake of example.
This example is also worth potentially building a tutorial video and sample front end for IMO.

@sunnyjaycer can you give this a look and let me know your thoughts?

sunnyjaycer commented:
Comments on README

Would be good to get a code snippet of the getTime usage described here:
- https://github.com/superfluid-finance/super-examples/tree/acl-example/projects/superfluid-gelato-automation#steps

1. Make it clearer somewhere here that we're doing this locally (not a testnet thing)
2. This has foundry usage, but guidance on it in the readme is rather scattered. Might be worth separating guidance on hardhat  and foundry
- https://github.com/superfluid-finance/super-examples/tree/acl-example/projects/superfluid-gelato-automation

...Will follow up with feedback after following instructions and running the code!

saflamini commented:
@sunnyjaycer - fixed make file, there was an issue with running it (at least for me locally)

Added a link to tutorial on using superfluid with hardhat and foundry
Added code example of using javascript date and time

sunnyjaycer commented:
`make` can be very difficult for Windows users. I encourage pointing towards the underlying sequence of commands for Windows folks in the README.

The deploy script here won't work locally unless you pass in parameters in an env which isn't mentioned in the README. Also, wouldn't you want to deployFrameworkAndTokens so you have the appropriate addresses to do that in the first place?
